src/app/services/actions/permission-mode-action-creator.ts

```diff
@@ -10,3 +12,17 @@ export function changeMode(newPerm: PERMISSION_MODE_TYPE): IAction {
         response: newPerm,
     };
 }
+
+export function closePopUp(response: boolean): IAction {
+    return {
+        type: CLOSE_POP_UP_SUCCESS,
+        response,
+    };
+}
+
+export function openPopUp(response: boolean): IAction {
+    return {
+        type: PREPARE_NEXT_POP_UP_SUCCESS,
+        response,
+    };
+}
```

Comment:
These are also the exact same function (code duplication), but with different action types. Might make sense to just have one action type called `togglePopUp`, similar to `src/app/services/actions/permissions-panel-action-creator.ts`.

Reply:
97d5789597b77a9f4963ddd78326fb10af34b004